WcaOnRails/spec/controllers/polls_controller_spec.rb

```diff
@@ -18,6 +18,24 @@
     end
   end
 
+  context "logged in as an admin" do
+    sign_in { FactoryBot.create :admin }
+    it "shows poll results" do
+      poll = FactoryBot.create(:poll)
+      get :results, params: { id: poll.id }
+      expect(response).to render_template("results")
+    end
+  end
+
+  context "logged in as a delegate" do
+    sign_in { FactoryBot.create :delegate }
+    it "shows poll results" do
+      poll = FactoryBot.create(:poll)
+      get :results, params: { id: poll.id }
+      expect(response).to render_template("results")
```

Comment:
nit: Not important, but I think we tend to use symbols here, so something like `render_template :results` instead.